inputs/text.vue

```diff
@@ -94,6 +96,9 @@
       },
       isInvalid() {
         return !!this.errorMessage;
+      },
+      numOfRows() {
+        return _.has(this.args,'rows') ? this.args.rows : 2;
```

Comment:
ah, it might be easier to do `_.get(this, 'args.rows', 2)`

Reply:
good point changing.